kube/smoke-test-list.yaml

```diff
@@ -0,0 +1,64 @@
+---
+kind: List
+apiVersion: v1
+metadata: {}
+items:
+- apiVersion: v1
+  kind: Pod
+  metadata:
+    name: origin-web-console-smoke-test-pod
+    labels:
+      app: origin-web-console-smoke-test
+    annotations:
+      prometheus.io/scrape: "true"
+      # todo: enable HTTPS
+      prometheus.io/scheme: http
+  spec:
+    restartPolicy: Never
+    containers:
+    - name: origin-web-console-smoke-test
+      image: benjaminapetersen/origin-web-console-smoke-test:latest
+      ports:                              
+      - containerPort: 3000
+        protocol: TCP
+      imagePullPolicy: IfNotPresent
+      env:
+      - name: CONSOLE_URL
+        value: https://192.168.100.3:8443
+- kind: Service
+  apiVersion: v1
+  metadata:
+    name: origin-web-console-smoke-test-service
+  spec:
+    ports:
+    - name: web
+      protocol: TCP
+      port: 3000
+      targetPort: 3000
+      nodePort: 0
+    selector:
+      app: origin-web-console-smoke-test
+    type: ClusterIP
+    clusterIP: 172.30.136.123
+    sessionAffinity: None
+  status:
+    loadBalancer: {}
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    name: run-protractor
+  spec:
+    schedule: "*/5 * * * *"
+    jobTemplate:
+      spec:
+        template:
+          metadata:
+            name: run-protractor
+            labels:
+              app: origin-web-console-smoke-test
+          spec:
+            containers:
+            - name: run-protractor
+              image: byrnedo/alpine-curl
+              command: ["/bin/sh",  "-c", "curl -k 172.30.136.123:3000/protractor"]
```

Comment:
Should we comment this as well, just to make sure the IPs match as we edit?  

Its prob not a huge deal, just thinking as I've had to `cluster down` and `cluster up` whenever I go to work or go home because my machine IP changes.

Reply:
Agreed, will add the comment :+1: